test/unit_tests/modules/FlyoutMenu-spec.js

```diff
@@ -0,0 +1,449 @@
+
+      expect( triggerClickSpy.callCount ).to.equal( 1 );
+      args = triggerClickSpy.getCall( 0 ).args[0];
+      expect( args.target ).to.equal( flyoutMenu );
+      expect( args.type ).to.equal( 'triggerClick' );
+    } );
+
+    it( 'should dispatch events when called by trigger click', function() {
+      var mouseEvent = document.createEvent( 'MouseEvents' );
+      mouseEvent.initEvent( 'mouseover', true, true );
+      triggerDom.dispatchEvent( mouseEvent );
+      triggerDom.click();
+    } );
+
+    xit( 'should dispatch events when called by alt trigger click', function() {
+      // TODO: alt trigger doesn't dispatch mouseover events,
+      //       but it probably should to match the trigger API.
+      var mouseEvent = document.createEvent( 'MouseEvents' );
+      mouseEvent.initEvent( 'mouseover', true, true );
+      altTriggerDom.dispatchEvent( mouseEvent );
+      altTriggerDom.click();
+    } );
+  } );
+
+  describe( '.expand()', function() {
+    beforeEach( function() {
+      // Set up expected event listeners.
+      expandBeginSpy = sinon.spy();
+      expandEndSpy = sinon.spy();
+
+      flyoutMenu.init();
+      flyoutMenu.addEventListener( 'expandBegin', expandBeginSpy );
+      flyoutMenu.addEventListener( 'expandEnd', expandEndSpy );
+    } );
+
+    afterEach( function() {
+      // Check expected event broadcasts.
+      expect( expandBeginSpy.callCount ).to.equal( 1 );
+      var args = expandBeginSpy.getCall( 0 ).args[0];
+      expect( args.target ).to.equal( flyoutMenu );
+      expect( args.type ).to.equal( 'expandBegin' );
+
+      expect( expandEndSpy.callCount ).to.equal( 1 );
+      args = expandEndSpy.getCall( 0 ).args[0];
+      expect( args.target ).to.equal( flyoutMenu );
+      expect( args.type ).to.equal( 'expandEnd' );
+
+      // Check expected aria-expanded state.
+      expect( triggerDom.getAttribute( 'aria-expanded' ) ).to.equal( 'true' );
+      expect( contentDom.getAttribute( 'aria-expanded' ) ).to.equal( 'true' );
+      expect( altTriggerDom.getAttribute( 'aria-expanded' ) )
+        .to.equal( 'true' );
+    } );
+
+    it( 'should dispatch events and set aria-expanded, ' +
+        'when called by trigger click', function() {
+      triggerDom.click();
+    } );
+
+    it( 'should dispatch events and set aria-expanded, ' +
+        'when called by alt trigger click', function() {
+      altTriggerDom.click();
+    } );
+
+    it( 'should dispatch events and set aria-expanded, ' +
+        'when called directly', function() {
+      flyoutMenu.expand();
+    } );
+  } );
+
+  describe( '.collapse()', function() {
+    beforeEach( function() {
+      // Set up expected event listeners.
+      collapseBeginSpy = sinon.spy();
+      collapseEndSpy = sinon.spy();
+
+      flyoutMenu.init();
+      flyoutMenu.addEventListener( 'collapseBegin', collapseBeginSpy );
+      flyoutMenu.addEventListener( 'collapseEnd', collapseEndSpy );
+      triggerDom.click();
```

Comment:
Yeah, let's get the acceptance test coverage in first and then refactor here to add the option to be expanded at initialization since none of the current flyoutmenus are open by default, but the expandables could use it. Todo added in https://github.com/cfpb/cfgov-refresh/pull/1724/commits/c75fd4e8304115bc2345f6b3449b882cb3856578